app/src/shared/components/Layout/Nav.styles.tsx

```diff
@@ -0,0 +1,368 @@
+    opacity: 1;
+`
+const CaretUpIcon = styled(SvgIcon)`
+    opacity: 0;
+`
+const DropdownToggle = styled.div`
+    background: #f8f8f8;
+    width: 32px;
+    height: 32px;
+    border-radius: 4px;
+    position: relative;
+    margin-top: 1px;
+
+    svg {
+        position: absolute;
+        left: 50%;
+        top: 50%;
+        transform: translate(-50%, -50%);
+        width: 10px;
+        height: 10px;
+        transition: 200ms opacity;
+    }
+`
+export const Menu = styled(UnstyledMenu)``
+export const MenuItem = styled(Menu.Item)`
+    &.user-info {
+        padding: 0 16px !important;
+    }
+    &.disconnect {
+        padding: 0 !important;
+        .disconnect-text {
+            padding: 12px 16px;
+            display: flex;
+            align-items: center;
+            justify-content: space-between;
+        }
+    }
+`
+export const MenuDivider = styled(Menu.Divider)`
+    margin: 0;
+`
+export const WalletAddress = styled.div`
+    margin-left: 13px;
+    display: flex;
+    flex-direction: column;
+
+    span {
+        font-size: 14px;
+        line-height: 18px;
+        user-select: none;
+        color: ${COLORS.primary};
+        font-weight: 400;
+
+        &.ens-name {
+            font-weight: 500;
+        }
+    }
+`
+export const SignedInUserMenu = styled(NavDropdown)`
+    ${Menu} {
+        width: 260px;
+        padding: 0;
+
+        ${Menu.Item}:first-child {
+            padding: 0 4px;
+            margin-bottom: 10px;
+        }
+
+        ${Avatarless} {
+        }
+
+        ${Name},
+        ${Username} {
+            overflow: hidden;
+            white-space: nowrap;
+            text-overflow: ellipsis;
+        }
+    }
+
+    :hover ${DropdownToggle} {
+        ${CaretDownIcon} {
+            opacity: 0;
+        }
+
+        ${CaretUpIcon} {
+            opacity: 1;
+        }
+    }
+`
+export const Navbar = styled.div`
+    display: grid;
+    grid-template-columns: auto 1fr auto auto;
+    align-items: center;
+`
+export const MenuGrid = styled.div`
+    display: grid;
+    grid-template-columns: auto auto auto auto;
+    justify-content: center;
+    align-items: center;
+`
+export const NavLink = styled.a``
+export const NavbarItem = styled.div`
+    ${MenuGrid} & + & {
+        margin-left: 16px;
+    }
+`
+const LinkWrapper = styled.div`
+    ${NavLink} {
+        display: block;
+        color: ${COLORS.primaryLight};
+        text-transform: uppercase;
+        font-weight: ${MEDIUM};
+        letter-spacing: 2px;
+        white-space: nowrap;
+        text-decoration: none !important;
+    }
+
+    &:hover {
+        ${NavLink} {
+            color: ${COLORS.primary};
+        }
+    }
+`
+type UnstyledNavbarLinkProps = {
+    highlight?: boolean
+    children: any
+}
+const UnstyledNavbarLink: FunctionComponent<UnstyledNavbarLinkProps> = ({
+    highlight,
+    children,
+    ...props
+}) => {
+    return <LinkWrapper {...props}>{children}</LinkWrapper>
+}
```

Comment:
TBH it's not even a part of what I was doing - you see it as changes because I moved the styles to a separate file, but of course I will implement the suggested changes :)

Reply:
also I do not actually get what you mean here and now when I look at it I think that best would be to remove this prop from the props type, it's being defined later by the "styled" component

Reply:
Yeah, it'd be totally fine (better?) to style `LinkWrapper` right away, directly.

And about naming props see https://styled-components.com/docs/api#transient-props.